activerecord/lib/active_record/connection_handling.rb

```diff
@@ -180,12 +180,14 @@ def connected_to(database: nil, role: nil, shard: nil, prevent_writes: false, &b
     #
     # Usage:
     #
-    #   ActiveRecord::Base.connected_to(AnimalsRecord, MealsRecord], role: :reading) do
+    #   ActiveRecord::Base.connected_to([AnimalsRecord, MealsRecord], role: :reading) do
```

Comment:
Shouldn't we rather promote the syntax without brackets in the documentation? Also, shouldn't it be `connected_to_many` rather than `connected_to`?

Reply:
(Rah, @eugeneius was faster. 🙊)

Reply:
![image](https://user-images.githubusercontent.com/432189/98046764-ae7bc380-1e22-11eb-995d-14e0f0e51ade.png)